src/main/scala/is/hail/io/vcf/ExportVCF.scala

```diff
@@ -232,15 +232,28 @@ object ExportVCF {
       } else
         TStruct.empty()
     
+    val gr = vsm.vSignature.asInstanceOf[TVariant].gr.asInstanceOf[GenomeReference] 
+    
     val localNSamples = vsm.nSamples
     val hasSamples = localNSamples > 0
 
     def header: String = {
       val sb = new StringBuilder()
 
       sb.append("##fileformat=VCFv4.2\n")
-      sb.append(s"##hailversion=${hail.HAIL_PRETTY_VERSION}\n")
-
+      sb.append(s"##hailversion=${ hail.HAIL_PRETTY_VERSION }\n")
+      sb.append(s"##reference=${ gr.name }\n")
```

Comment:
I think this should be in each contig line with the key `assembly` rather than as a separate field in the header. This is the example Yossi gave:

`##contig=<ID=20,length=62435964,assembly=B36,md5=f126cdf8a6e0c7f379d618ff66beb2da>`